src/autotype/AutoType.cpp

```diff
@@ -342,6 +355,13 @@ QList<AutoTypeAction*> AutoType::createActionFromTemplate(const QString& tmpl, c
     int num = -1;
     QList<AutoTypeAction*> list;
 
+    QRegExp delayRegEx("delay=(\\d+)", Qt::CaseSensitive, QRegExp::RegExp2);
```

Comment:
A couple lines up there is a conversion to lowercase. It caught me off guard too, but the other regex in the file are the same way.